be/src/exprs/like_predicate.cpp

```diff
@@ -49,7 +49,9 @@ bool LikePredicate::hs_compile_and_alloc_scratch(const std::string& pattern, Lik
     if (hs_compile(pattern.c_str(), HS_FLAG_ALLOWEMPTY | HS_FLAG_DOTALL | HS_FLAG_UTF8 | HS_FLAG_SINGLEMATCH,
                    HS_MODE_BLOCK, nullptr, &state->database, &state->compile_err) != HS_SUCCESS) {
         std::stringstream error;
-        error << "Invalid hyperscan expression: " << std::string(slice.data, slice.size) << ": "
+        auto chopped_size = std::min<size_t>(slice.size, 64);
+        auto ellipsis = (chopped_size < slice.size) ? "..." : "";
+        error << "Invalid hyperscan expression: " << std::string(slice.data, chopped_size) << ellipsis << ": "
```

Comment:
```suggestion
        error << "Invalid hyperscan expression: " << std::string_view(slice.data, chopped_size) << ellipsis << ": "
```

std::string_view might be lightweight in such case.

Reply:
it is a low-frequency executing code.